### ISSUE TITLE
Installer cmd line argument -uninstall  triggers installation

### ISSUE THREAD

**Issue by Palciny** [sumatrapdfreader/sumatrapdf#4601]
**SumatraPDF version**
- Version 3.5.2, pre-release

**Describe the bug**
Based on https://www.sumatrapdfreader.org/docs/Installer-cmd-line-arguments
Argument -uninstall : uninstalls SumatraPDF

When I run 
.\SumatraPDF-prerel-64-install.exe -uninstall
It just open installer

**To Reproduce**
1. Download Installer
2. Open powershell
3. .\SumatraPDF-prerel-64-install.exe -uninstall
4. See installation window open

**Expected behavior**
Uninstall windows should open

**File that reproduces the problem**
SumatraPDF-3.5.2-64-install.exe, SumatraPDF-prerel-64-install.exe installer

**Screenshots**
![image](https://github.com/user-attachments/assets/f7e0fd21-2e2b-4e6c-ba50-9dc3946252d2)

**Comment by GitHubRulesOK** [comment]
the issue is you are running the installer `-install.exe -uninstall` the whole concept of "one.exe" is the filename is the switch thus simply rename `-uninstall.exe` that should work ?

**Comment by GitHubRulesOK** [comment]
yes the doc is perhaps not specific that it is the **installed.exe** needs the uninstall command so for example

`"%programfiles%\sumatrapdf\SumatraPDF.exe" -uninstall`
